### PR TITLE
Add <get_performance>

### PR DESCRIPTION
**Pull request by jjnicola**
Allows to execute a external program/script and get the response.
The program must be named 'gvmcg' and should receive up to 3 arguments
called start, end and titles.

Example with gvm-cli:

```
gvm-cli --protocol OSP socket --socketpath=/run/ospd/openvas.sock --xml "<get_performance start='0' titles='titles'/>"

```

### CODE REVIEW

File: ospd/ospd.py [hunk 1]
```diff
@@ -978,6 +1007,62 @@ def handle_get_vts_command(self, vt_et):
...
+        """ Handles <get_performance> command.
+
+        @return: Response string for <get_performance> command.
+        """
+        start = scan_et.attrib.get('start')
+        end = scan_et.attrib.get('end')
+        titles = scan_et.attrib.get('titles')
+
+        cmd = list()
+        cmd.append('gvmcg')
```
Comment (bjoernricks) [inline ospd/ospd.py]: Can be sumarized in one line
```suggestion
        cmd = ['gvmcg']
```

File: ospd/ospd.py [hunk 2]
```diff
@@ -978,6 +1007,62 @@ def handle_get_vts_command(self, vt_et):
...
+    def handle_get_performance(self, scan_et):
+        """ Handles <get_performance> command.
+
+        @return: Response string for <get_performance> command.
+        """
+        start = scan_et.attrib.get('start')
+        end = scan_et.attrib.get('end')
+        titles = scan_et.attrib.get('titles')
+
+        cmd = list()
```
Comment (bjoernricks) [inline ospd/ospd.py]:
```suggestion
```

File: tests/test_scan_and_result.py [hunk 3]
```diff
@@ -229,6 +231,45 @@ def test_get_default_help(self):
...
+
+        self.assertEqual(response.get('status'), '200')
+        self.assertEqual(response.tag, 'get_performance_response')
+
+    def test_get_performance_fail_int(self):
+        daemon = DummyWrapper([])
+        cmd = secET.fromstring(
+            '<get_performance start="a" end="0" titles="mem"/>')
+
+        self.assertRaises(
```
Comment (bjoernricks) [inline tests/test_scan_and_result.py]: Personally I would use the with statement for assert raises (see https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises) but it's up to you :-)

Comment (jjnicola) [reply tests/test_scan_and_result.py]: Other tests don't use the with statement for assert raises neither. I will do this in a future PR.

**Pull request merged**